interface/repository/csv_item_repository.go

```diff
@@ -0,0 +1,62 @@
+package repository
+
+import (
+	"academy_bootcamp/application/contracts"
+	"academy_bootcamp/domain/model"
+	"academy_bootcamp/infrastructure/datastore"
+	"errors"
+	"io"
+	"strconv"
+)
```

Comment:
Imported packages should be organized into three groups:
1. Golang's libraries
2. Internal packages
3. Third-party packages
```suggestion
import (
	"errors"
	"io"
	"strconv"
	
	"academy_bootcamp/application/contracts"
	"academy_bootcamp/domain/model"
	"academy_bootcamp/infrastructure/datastore"
)
```